admin_manual/installation/nginx-root.conf.sample

```diff
@@ -57,7 +57,7 @@ server {
     gzip_comp_level 4;
     gzip_min_length 256;
     gzip_proxied expired no-cache no-store private no_last_modified no_etag auth;
-    gzip_types application/atom+xml application/javascript application/json application/ld+json application/manifest+json application/rss+xml application/vnd.geo+json application/vnd.ms-fontobject application/wasm application/x-font-ttf application/x-web-app-manifest+json application/xhtml+xml application/xml font/opentype image/bmp image/svg+xml image/x-icon text/cache-manifest text/css text/plain text/vcard text/vnd.rim.location.xloc text/vtt text/x-component text/x-cross-domain-policy;
+    gzip_types application/atom+xml text/javascript application/json application/ld+json application/manifest+json application/rss+xml application/vnd.geo+json application/vnd.ms-fontobject application/wasm application/x-font-ttf application/x-web-app-manifest+json application/xhtml+xml application/xml font/opentype image/bmp image/svg+xml image/x-icon text/cache-manifest text/css text/plain text/vcard text/vnd.rim.location.xloc text/vtt text/x-component text/x-cross-domain-policy;
```

Comment:
this only changes which mime have gzip enabled, right? Nextcloud still sends (some) files as application/javascript, e.g. in `\OC\Core\Controller\JsController::getJs` and `\OCA\Files\Controller\ApiController::serviceWorker`.

I'd propose to change the gzip config to compress both `text/javascript` **and** `application/javascript`.

Reply:
Yes I will adjust it to include both MIME types.

Reply:
Done :heavy_check_mark: